diff --git a/deps/cpu_features/src/cpuinfo_x86.c b/deps/cpu_features/src/cpuinfo_x86.c
index 18e88c1..e54edd6 100644
--- a/deps/cpu_features/src/cpuinfo_x86.c
+++ b/deps/cpu_features/src/cpuinfo_x86.c
@@ -102,7 +102,6 @@
     defined(CPU_FEATURES_OS_FREEBSD)
 #include "internal/filesystem.h"         // Needed to parse /proc/cpuinfo
 #include "internal/stack_line_reader.h"  // Needed to parse /proc/cpuinfo
-#include "internal/string_view.h"        // Needed to parse /proc/cpuinfo
 #elif defined(CPU_FEATURES_OS_DARWIN)
 #if !defined(HAVE_SYSCTLBYNAME)
 #error "Darwin needs support for sysctlbyname"
@@ -111,6 +110,7 @@
 #else
 #error "Unsupported OS"
 #endif  // CPU_FEATURES_OS
+#include "internal/string_view.h"
 
 ////////////////////////////////////////////////////////////////////////////////
 // Definitions for CpuId and GetXCR0Eax.
